dcc-portal-ui/app/scripts/genes/views/gene.html

```diff
@@ -47,7 +47,9 @@ <h1 data-ui-scrollfix=79 ><span class="t_badge t_badge__gene"></span>
                            data-facet-name="functionalImpact"
                            data-hide-count="true"
                            data-label="{{'Mutation Impact' | translate}}"
-                           data-facet="GeneCtrl.mutationFacets.functionalImpact"></terms>
+                           data-facet="GeneCtrl.mutationFacets.functionalImpact"
+                           data-reset-pagination-on-change="true"
```

Comment:
I assume you already did this analysis and that is why your PR is touching many cross-functional locations.